models/pod.go

```diff
@@ -296,16 +292,17 @@ func getContainerWithName(name string, pod *Pod) v1.Container {
 	return container
 }
 
+// configureHostPorts will fail if it cannot access redis.
 func (p *Pod) configureHostPorts(
 	configYaml *ConfigYAML,
-	clientset kubernetes.Interface,
 	redisClient redisinterfaces.RedisClient,
 	mr *MixedMetricsReporter,
 ) error {
 	pod, err := GetPodFromRedis(redisClient, mr, p.Name, p.Namespace)
 	if err != nil {
+		// TODO, FIXME(lhahn): this error is misleading.
```

Comment:
Is there a specific reason for not fixing it now and just leave the TODO instead?